docs/_posts/2016-09-27-leaflet-1.0-final.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: The wait is over. Meet Leaflet 1.0.
```

Comment:
This sounds so much like an apple commercial, I'll ask our in-house designer to throw some meme ideas.